src/components/Profile/Button/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+import { StyledButton } from './Button.styled.js';
+
+const Button = ({ title, ...rest }) => {
```

Comment:
It's redundant component. Make Styled button directly, without it

Reply:
delete